src/resolve/resolved-manager.c

```diff
@@ -565,6 +566,69 @@ static int manager_memory_pressure_listen(Manager *m) {
         return 0;
 }
 
+static int manager_dispatch_reload_signal(sd_event_source *s, const struct signalfd_siginfo *si, void *userdata) {
+        Manager *m = ASSERT_PTR(userdata);
+        DnssdService *dnssd;
+        int r;
+
+        (void) sd_notifyf(/* unset= */ false,
+                          "RELOADING=1\n"
+                          "STATUS=Reloading configuration...\n"
+                          "MONOTONIC_USEC=" USEC_FMT, now(CLOCK_MONOTONIC));
+
+        m->llmnr_support = DEFAULT_LLMNR_MODE;
+        m->mdns_support = DEFAULT_MDNS_MODE;
+        m->dnssec_mode = DEFAULT_DNSSEC_MODE;
+        m->dns_over_tls_mode = DEFAULT_DNS_OVER_TLS_MODE;
```

Comment:
IIRC, these mode can be also changed through DBus. Maybe we should keep them if they are changed by DBus?

Reply:
Only the per-interface modes can be set via D-Bus, the "global" ones cannot, they are only configured via the config file.